components/ActionsDropdown/model.ts

```diff
@@ -16,3 +18,11 @@ export interface IActionsDropdownItemComponentProps extends IBoxComponentProps {
 	/** value returned in callback fn */
 	value?: any;
 }
+
+export interface IActionsDropdownButtonCloseComponentProps extends IBoxComponentProps {
```

Comment:
not needed, Box already has onClick, value isnt needed, we're not returning anything from a click except the event.